cli/.gitignore

```diff
@@ -0,0 +1,134 @@
+# Logs
+logs
```

Comment:
we should clean it, looks like a boilerplate gitignore with a lot of folder we won't have!

Reply:
yes it was Github's boilerplate for node apps, I'll change it, thanks for the feedback!